pkg/disk/swap.go

```diff
@@ -0,0 +1,76 @@
+package disk
+
+import (
+	"math/rand"
+	"reflect"
+
+	"github.com/google/uuid"
+)
+
+// Swap defines the payload for a swap partition. It's similar to a
+// [Filesystem] but with fewer fields. It is a [PayloadEntity] and also a
+// [FSTabEntity].
+type Swap struct {
+	UUID  string
+	Label string
+
+	// The fourth field of fstab(5); fs_mntops
+	FSTabOptions string
+}
+
+func init() {
+	payloadEntityMap["swap"] = reflect.TypeOf(Swap{})
+}
+
+func (s *Swap) EntityName() string {
+	return "swap"
+}
+
+func (s *Swap) Clone() Entity {
+	if s == nil {
+		return nil
+	}
+
+	return &Swap{
+		UUID:         s.UUID,
+		FSTabOptions: s.FSTabOptions,
+	}
```

Comment:
`Label: s.Label,` is missing. Is this intentional? :thinking:

Reply:
Not intentional.  Thanks for catching it.

Reply:
Fixed.